filebeat/docs/modules/envoyproxy.asciidoc

```diff
@@ -0,0 +1,36 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-envoyproxy]]
+[role="xpack"]
+
+:modulename: envoyproxy
+:has-dashboards: true
+
+== Envoyproxy Module
+
+This is a filebeat module for envoy proxy. It supports both standalone deployment and 
+envoy proxy deployment in Kubernetes. 
+
+[float]
+=== Compatibility
+
+Although this module has been developed against Kubernetes v1.13.x, it is expected to work
```

Comment:
What about slicing this to?
```
This module has been developed against Kubernetes v1.13.x.
```
Would also be nice (but not needed) to add the Envoy version tested.
What do you think?

Reply:
ok, I commented on the generated file instead of the source :-1: